### DISCUSSION TITLE
How do I change menu items in the top left of the page

### DISCUSSION THREAD

**Discussion by armandjordaan-inovo** [super-productivity/super-productivity#3354] [Q&A] [answered]
The timeline menu item has disappeared for me and I don't know how to get it back. Is there a settings that I can change?

This is how it used to be:
![Screenshot_20240819_034923](https://github.com/user-attachments/assets/df12ecd8-27a7-4170-8440-305439ff4aa3)

This is the menu I see now, but I don't know how to change it back.
![Screenshot_20240819_034956](https://github.com/user-attachments/assets/1518eed7-b7f4-478b-b1df-a47e08a16e43)

**Comment by johannesjo** [comment]
It seems like you installed SP from the edge channel. The Navigation will be updated and timeline will be replaced by schedule. "Scheduled" is now sligthly different and called "Repeat/Scheduled" and move further down the navigation.

**Comment by armandjordaan** [comment]
Does that mean the "Timeline" view will not be available any longer? That will be a real pity since I find it an extremely useful view, much more useful that the "Planner" or "Schedule" views for me.

**Reply by johannesjo** [reply, marked as answer]
No the timeline will not be available anymore. Imho the schedule feature does something very similar but better.

But you can up a feature request to restore the original timeline. If many people want it, I'll bring it back :)

**Comment by armandjordaan-inovo** [comment]
After living with "Scheduler" and "Planner" for a few days, I think you may be right, they are quite useful for planning one's week. I'll just use them instead of the "Timeline" view.
